src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -615,12 +611,17 @@ std::shared_ptr<GLHEVertArray> GetVertArray(EnergyPlusData &state, std::string c
 std::shared_ptr<GLHEResponseFactors> GetResponseFactor(EnergyPlusData &state, std::string const &objectName)
 {
     // Check if this instance of this model has already been retrieved
-    for (auto &thisRF : state.dataGroundHeatExchanger->responseFactorsVector) {
-        // Check if the type and name match
-        if (objectName == thisRF->name) {
-            return thisRF;
-        }
-    }
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:620]:(style),[useStlAlgorithm],Consider using std::find_if algorithm instead of a raw loop.